BlueprintUI/Sources/Layout/EqualSizeStack.swift

```diff
@@ -0,0 +1,134 @@
+/// An element that sizes all of its children equally, stacking them according to
+/// the specified `direction` and spacing them according to the specified `gutter`.
+public struct EqualSizeStack: Element {
+
+    /// The direction in which this element will stack its children.
+    public var direction: Direction
+
+    /// The amount of space between children in this element.
+    public var gutter: CGFloat
```

Comment:
For consistency with `Stack`, I would call this `spacing` (`Stack` uses `minimumSpacing` as it can expand the spacing if needed)